src/odemis/acq/drift/test/drift_test.py

```diff
@@ -27,14 +29,86 @@
 DATA_DIR = os.path.dirname(__file__)
 
 
+
+CONFIG_SED = {"name": "sed", "role": "sed"}
+CONFIG_SCANNER = {"name": "scanner", "role": "ebeam"}
+CONFIG_SEM = {"name": "sem", "role": "sem", "image": "simsem-fake-output.h5",
+              "drift_period": 0.1,
+              "children": {"detector0": CONFIG_SED, "scanner": CONFIG_SCANNER}
+              }
+
 class TestAnchoredEstimator(unittest.TestCase):
     """
     Test AnchoredEstimator
     """
 
+    @classmethod
+    def setUpClass(cls):
+        cls.sem = simsem.SimSEM(**CONFIG_SEM)
+
+        for child in cls.sem.children.value:
+            if child.name == CONFIG_SED["name"]:
+                cls.detector = child
+            elif child.name == CONFIG_SCANNER["name"]:
+                cls.scanner = child
+
+        # cls.roi = model.TupleContinuous(UNDEFINED_ROI,
+        #                                  range=((0, 0, 0, 0), (1, 1, 1, 1)),
+        #                                  cls=(int, long, float))
+        #                                  # setter=self._setROI)
+        # cls.dwellTime = model.FloatContinuous(cls.scanner.dwellTime.range[0],
+        #                                        range=cls.scanner.dwellTime.range, unit="s")
+
+        cls.region = (0, 0, 0.1, 0.1)
+        cls.dwellTime = 5e-6
```

Comment:
The `setUpclass` is only executed at the start of running the entire test class. You cannot be sure if the test cases are always executed in the same order, therefore if `self.region` or `self.dwellTime` is changed anywhere in a test case you might not have the same input when you rerun all the test cases. So I would suggest to set theses values in the test cases it self, or add a `setUp` and put it there. See for instance `odemis.driver.test.scanner_test.TestDetectorHw.setUp`